reference/commands/misc/lock.rst

```diff
@@ -0,0 +1,143 @@
+                            Provide a package reference instead of a conanfile
+      -l LOCKFILE, --lockfile LOCKFILE
+                            Path to lockfile to be used as a base
+      --base                Lock only recipe versions and revisions
+      --lockfile-out LOCKFILE_OUT
+                            Filename of the created lockfile
+      -b [BUILD], --build [BUILD]
+                            Packages to build from source
+      -r REMOTE, --remote REMOTE
+                            Look in the specified remote server
+      -u, --update          Will check the remote and in case a newer version and/or revision of the dependencies exists there, it will install those in the local cache. When
+                            using version ranges, it will install the latest version that satisfies the range. Also, if using revisions, it will update to the latest revision
+                            for the resolved version range.
+      -e ENV_HOST, --env ENV_HOST
+                            Environment variables that will be set during the package build (host machine). e.g.: -e CXX=/usr/bin/clang++
+      -e:b ENV_BUILD, --env:build ENV_BUILD
+                            Environment variables that will be set during the package build (build machine). e.g.: -e:b CXX=/usr/bin/clang++
+      -e:h ENV_HOST, --env:host ENV_HOST
+                            Environment variables that will be set during the package build (host machine). e.g.: -e:h CXX=/usr/bin/clang++
+      -o OPTIONS_HOST, --options OPTIONS_HOST
+                            Define options values (host machine), e.g.: -o Pkg:with_qt=true
+      -o:b OPTIONS_BUILD, --options:build OPTIONS_BUILD
+                            Define options values (build machine), e.g.: -o:b Pkg:with_qt=true
+      -o:h OPTIONS_HOST, --options:host OPTIONS_HOST
+                            Define options values (host machine), e.g.: -o:h Pkg:with_qt=true
+      -pr PROFILE_HOST, --profile PROFILE_HOST
+                            Apply the specified profile to the host machine
+      -pr:b PROFILE_BUILD, --profile:build PROFILE_BUILD
+                            Apply the specified profile to the build machine
+      -pr:h PROFILE_HOST, --profile:host PROFILE_HOST
+                            Apply the specified profile to the host machine
+      -s SETTINGS_HOST, --settings SETTINGS_HOST
+                            Settings to build the package, overwriting the defaults (host machine). e.g.: -s compiler=gcc
+      -s:b SETTINGS_BUILD, --settings:build SETTINGS_BUILD
+                            Settings to build the package, overwriting the defaults (build machine). e.g.: -s:b compiler=gcc
+      -s:h SETTINGS_HOST, --settings:host SETTINGS_HOST
+                            Settings to build the package, overwriting the defaults (host machine). e.g.: -s:h compiler=gcc
+
+
+conan lock update
+-----------------
+
+.. code-block:: bash
+
+  $ conan lock update [-h] old_lockfile new_lockfile
```

Comment:
make sure this is rendering correctly, this indentation seems wrong. I'd prefer to use always 4 spaces

Reply:
Yes, it is rendering ok, but agree, I will change it